data/systemd/snapd.run-from-snap

```diff
@@ -34,13 +34,29 @@ case $CMD in
             echo "cannot run without the prefix for the snapd snap"
             exit 1
         fi
+        echo "Add the bind mount unit of the snapd tooling"
+        cat >/run/systemd/system/usr-lib-snapd.mount <<EOF
+[Unit]
+Description=Make the snapd snap tooling available for the system
+Before=snapd.service
+
+[Mount]
+What=$PREFIX/usr/lib/snapd
+Where=/usr/lib/snapd
```

Comment:
Do we need to take libexec into account?

Reply:
Thanks for checking. This code only runs on ubuntu-core systems, so we can hardcode the (known) libexecdir here.

Reply:
So then we'll have to fix this when there's more than Ubuntu Core systems? :/